### ISSUE TITLE
Wiki as git sub-module

### ISSUE THREAD

**Issue by tonytheodore** [mxe/mxe#1271]
There's some very useful documentation hidden in https://github.com/mxe/mxe/wiki and I'd like to make this available in the main tree (with the longer term aim of moving all docs to wiki/markdown style and reserve `index.html` for "website" info - intro, contacts, external links etc).

Does anyone have experience with keeping these in sync?

**Comment by starius** [comment]
```
git submodule add https://github.com/mxe/mxe.wiki.git doc/wiki
```

I think, it is better to keep all the information in the repo itself. For example, if you change something, you can (and should) change the documentation of this feature in the same commit.

**Comment by tonytheodore** [comment]
Indeed, actual content is more important than clever ways to format and sync it.

**Comment by starius** [comment]
I want to revisit this issue. Page [Add a New Package: Full Version](https://github.com/mxe/mxe/wiki/Add-a-New-Package:-Full-Version) seems to have an outdated template in the bottom that [caused confusion](https://github.com/mxe/mxe/pull/1591#discussion_r92616317).

Can we update all wiki pages, then move them to mxe/docs and then remove the wiki?
